src/charts/gv-chart-line.js

```diff
@@ -59,6 +59,19 @@ export class GvChartLine extends ChartElement(LitElement) {
         }
       });
     }
+    this.options.plotLines = this.options.plotLines ? this.options.plotLines.map((line) => {
+      return {
+        ...line,
+        events: {
+          mouseover: function () {
+            this.label.element.firstElementChild.style.visibility = 'visible';
```

Comment:
Is `firstElementChild` really needed? In my test cases updating the style of `element` directly was working too 🤷🏻

Reply:
Yes, otherwise it is not working in APIM UI